Makefile

```diff
@@ -2,7 +2,7 @@
 
 IMAGE := $(or ${IMAGE},quay.io/ocpmetal/ocp-metal-ui:latest)
 TESTS_IMAGE := $(or ${TESTS_IMAGE},quay.io/ocpmetal/ocp-metal-ui-tests:latest)
-BM_INVENTORY_URL := $(or ${BM_INVENTORY_URL},http://bm-inventory.assisted-installer.svc.cluster.local:8090)
+ASSISTED_SERVICE_URL := $(or ${ASSISTED_SERVICE_URL},http://assisted-service.assisted-installer.svc.cluster.local:8090)
```

Comment:
mm NAMESPACE? instead of assisted-installer?

Reply:
it's a different bug but i can probably fix it

Reply:
:)

Reply:
fixed